.github/workflows/test-linux-distros.yaml

```diff
@@ -44,7 +44,9 @@ jobs:
           });
 
           // Output a JSON object consumed by the build matrix below.
-          console.log(`::set-output name=MATRIX::${ JSON.stringify(matrix) }`);
+          fs.writeFileSync(process.env.GITHUB_OUTPUT,
+              `MATRIX=${ JSON.stringify(matrix) }`,
+              {flag: 'a'});
```

Comment:
OOC, what does "flag: 'a'" mean? append?

Reply:
Yes, append.  It's the nodejs fs API's equivalent of `fopen(..., "a")`